### DISCUSSION TITLE
Recommendation on custom css usage

### DISCUSSION THREAD

**Discussion by rmatuszczakp** [twilio-labs/paste#3277] [General] [answered]
Hello! 
I would like to learn about the recommended approach to using custom css when a specific design doesn't exist within the paste design system. In on of our projects we have a box that we would like to have a gradient background which doesn't seem to exist in Paste. What would be your recommended solution for Twilio apps in such a case? Should we push back with the design proposal and stick to what is available within Paste? Should we somehow request for the gradient property to be made? Or should we go ahead and implement it, and if yes what would be the best approach? 

Please take a look at the following PR that has a screenshot of the box: https://github.com/twilio-internal/console/pull/5010 

Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hi there

I created a Codesandbox in the duplicate discussion thread about this. You should just be able to copy it. 

https://codesandbox.io/s/gradient-bar-example-fzs2zc?file=/src/index.tsx
